Text To Speech/Text-to-Speech-Tutorial.md

```diff
@@ -127,4 +127,4 @@ file_name ="text_to_speeh_pitch.wav"
 
 #### Conclusion:
 
-This tutorial walks you through the steps of starting a Text to Speech service on the kube cluster, processing the dataset and using the Text to Speech service. This tutorial also shows you how to modify speech synthesis characteristics using different parameters.
\ No newline at end of file
+This tutorial walks you through the steps of starting a Text to Speech service on the kube cluster, processing the dataset and using the Watson NLP Text to Speech service. This tutorial also shows you how to modify speech synthesis characteristics using different parameters.
```

Comment:
This is not called Watson NLP so we should name it Watson Text to Speech or something similar